src/index.css

```diff
@@ -1,21 +0,0 @@
-* {
```

Comment:
why did you delete this file?

Reply:
I left it empty therefore git didn't add it. I added it back a few minutes ago.